PROJECT_PROGRESSION.md

```diff
@@ -16,14 +16,59 @@ This governance policy sets forth the proposal process for projects to be accept
 
 New projects should express interest to join the OpenJS Foundation via an email to new-projects@lists.openjsf.org with a filled out [Project Application Template](./NEW_PROJECT_APPLICATION.md). The initial application will have a silent period for internal only discussion. If your application is denied during this phase you will be notified privately. If there are no objections to the application, projects will given an `incubation` status and begin the on-boarding process. **Note** the `incubation` status is temporary as we determine if the project is a good fit. At any time during the `incubation` process a project can withdraw either voluntarily or at the request of the CPC.
 
-#### Application Phases
+#### Roles
+
+##### Applicant
+
+Individual or group who submits the application on behalf of a project to join the OpenJS Foundation.
+
+##### Application Manager
+
+The application manager ensures progress through the process, helps the project understand the work to be completed and
+to move though the application process. Typically the application manager will be one of the Foundation staff members.
+
+##### Application Champion
+
+The application champion is a member of the CPC who commits to do the following:
+
+1. Provide weekly update to the applicant on progression through the process.
+1. Be available to answer questions from the applicant.
+1. Advocate on behalf of the project applying.
+1. Help to explain why the project is a good fit.
+1. Help the application manager when needed.
+
+
+#### Process
 
 1. Initial email sent to new-projects@lists.openjsf.org with filled out [Project Application Template](./NEW_PROJECT_APPLICATION.md).
+1. An acknowledgement is sent to the application by Foundation staff within 48 hours (striving to reply within 24hours when possible).
 1. Silent period. Internal only discussion.
-1. Initial acceptance as [Incubating][] Project.
+1. Initial acceptance as [Incubating][] Project. Before public announcement there will be:
+   * Go/no go confirmation to the applicant that they are ready to have their application become public.
```

Comment:
I'm confused about the process here.

From the perspective of an applicant, what would be ideal is to get the below mail sent and collect all of the CPC member approval, and then get back to the applicant with the go-ahead to sync on an announcement date.

The sync bit is particularly important for larger projects.

Reply:
Flipped the order and added a line about agreeing on an announce date.